### PR TITLE
Rewrite the Calculator Docs and Fix ExNihilo

### PR DESCRIPTION
**Pull request by CrankySupertoon**
Seriously Why was the Calculator PR Merged

### CODE REVIEW

**Comment by CrankySupertoon** [comment]
#254 and #255 Should of never been merged @jaredlll08, they are missing in the mkdocs yaml file.

And #254 was just so mess i seperated all the machines instead of having like 8 machines in misc

File: docs/Mods/Calculator/Basic_Calculator.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Basic Calculator
+
+## Package
+```zenscript
+import mods.calculator.basic;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input[1-4]** The inputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Basic_Calculator.md]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input[1-2]** The inputs.
```

File: docs/Mods/Calculator/Processing_Chamber.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Package
+```zenscript
+import mods.calculator.processingChamber;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output[1-2]** The outputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Processing_Chamber.md]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output1** The output.
```

File: docs/Mods/Calculator/Reassembly_Chamber.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Package
+```zenscript
+import mods.calculator.reassemblyChamber;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output[1-2]** The outputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Reassembly_Chamber.md]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output1** The output.
```

File: docs/Mods/Calculator/Restoration_Chamber.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Package
+```zenscript
+import mods.calculator.restorationChamber;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output[1-2]** The outputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Restoration_Chamber.md]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output1** The output.
```

File: docs/Mods/Calculator/Scientific_Calculator.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Scientific Calculator
+
+## Package
+```zenscript
+import mods.calculator.scientific;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input[1-4]** The inputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Scientific_Calculator.md]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input[1-2]** The inputs.
```

File: docs/Mods/Ex_Nihilo_Creatio/Ores.md [hunk 6]
```diff
@@ -1,10 +1,9 @@
-# Ore
+# Ores
 
 ## Package
-
+```zenscript
```
Comment (kindlich) [inline docs/Mods/Ex_Nihilo_Creatio/Ores.md]: If you put the package name in a codeblock, then the Backticks around the package name would need to be removed.

However, in the other files from the ExnihiloCreatio documentation this is not done, so please try to be consistent within one mod's documentation.

File: docs/Mods/Ex_Nihilo_Creatio/Sieving.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# Sieving
```
Comment (kindlich) [inline docs/Mods/Ex_Nihilo_Creatio/Sieving.md]: Why does this show as a new file, when you only renamed it? :thinking:

File: mkdocs.yml [hunk 8]
```diff
@@ -378,57 +396,64 @@ pages:
...
-      - Tinkers' Construct:
-        - Brackets:
-          - Materials: 'Mods/ContentTweaker/Tinkers_Construct/Brackets/Bracket_Material.md'
-          - Traits: 'Mods/ContentTweaker/Tinkers_Construct/Brackets/Bracket_Trait.md'
-        - Material: 'Mods/ContentTweaker/Tinkers_Construct/Material.md'
-        - Material Builder: 'Mods/ContentTweaker/Tinkers_Construct/MaterialBuilder.md'
-        - Trait: 'Mods/ContentTweaker/Tinkers_Construct/Trait.md'
-        - Trait Builder: 'Mods/ContentTweaker/Tinkers_Construct/TraitBuilder.md'
-        - Trait Data Representation: 'Mods/ContentTweaker/Tinkers_Construct/TraitDataRepresentation.md'
+        - Material System:
```
Comment (kindlich) [inline mkdocs.yml]: Why did you put them all below CoT/Vanilla?
They are supposed to stay where they are!
Vanilla, MaterialSystem, TinkersConstruct and Chickens are the 4 Modules CoT offers, so it does not make sense to put three of them inside the forth.

File: docs/Mods/Calculator/Redstone_Extractor.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+## Package
+```zenscript
+import mods.calculator.redstone;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **Int value** The amount of input adds to that machine.
```
Comment (kindlich) [inline docs/Mods/Calculator/Redstone_Extractor.md]: This is not a grammatically correct construct.

File: docs/Mods/Calculator/Starch_Extractor.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+## Package
+```zenscript
+import mods.calculator.starch;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **Int value** The amount of input adds to that machine.
```
Comment (kindlich) [inline docs/Mods/Calculator/Starch_Extractor.md]: This is not a grammatically correct construct.

**Comment by jaredlll08** [comment]
> #254 and #255 Should of never been merged @jaredlll08, they are missing in the mkdocs yaml file.
> 
> And #254 was just so mess i seperated all the machines instead of having like 8 machines in misc

For the future, if something shouldn't be merged, the PR should be marked as a draft.

**Comment by kindlich** [comment]
@CrankySupertoon 
Could you merge the current master branch into this PR so that the Travis build can run with the updated settings?

There were quite some changes, so currently the CI doesn't even start properly for your build ^^'

File: docs/Mods/Calculator/Atomic_Calculator.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
+# Atomic Calculator
+
+## Package
+```zenscript
+import mods.calculator.atomic;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input[1-4]** The inputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Atomic_Calculator.md:10]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input[1-3]** The inputs.
```

File: docs/Mods/Calculator/Fabrication_Chamber.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+## Not fully implimented, only takes one input.
+
+## Package
+```zenscript
+import mods.calculator.fabricationChamber;
+```
+
+## Methods
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output[1-2]** The outputs.
```
Comment (kindlich) [inline docs/Mods/Calculator/Fabrication_Chamber.md:11]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output1** The outputs.
```

File: docs/Mods/Calculator/Glowstone_Extractor.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+## Package
+```zenscript
+import mods.calculator.glowstone;
+```
+
+## Methods
+
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** The input.
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) output** The output.
```
Comment (kindlich) [inline docs/Mods/Calculator/Glowstone_Extractor.md:11]:
```suggestion
```

File: docs/Mods/Ex_Nihilo_Creatio/Ores.md [hunk 14]
```diff
@@ -1,10 +1,9 @@
...
 `mods.exnihilocreatio.Ore`
-
-Also needs to be in the `#loader preinit` or `#loader contenttweaker`
```
Comment (kindlich) [inline docs/Mods/Ex_Nihilo_Creatio/Ores.md:7]: If that is still the case, please leave it in

**Comment by CrankySupertoon** [comment]
#273 #274

**Pull request closed without merging**